src/dex/generic-rfq/rate-fetcher.ts

```diff
@@ -200,17 +200,12 @@ export class RateFetcher {
   private handlePairsResponse(resp: PairsResponse) {
     this.pairs = {};
 
-    if (this.rateFetcher.isPolling()) {
-      this.rateFetcher.stopPolling();
-    }
-
     const pairs: PairMap = {};
     for (const pairName of Object.keys(resp.pairs)) {
       pairs[pairName] = resp.pairs[pairName];
     }
 
     this.pairs = pairs;
-    this.rateFetcher.startPolling();
```

Comment:
I think it's still necessary that `this.rateFetcher` is started otherwise it disables Generic RFQ.

Reply:
@skypper I applied new changes since your comment. Now rate fetcher polling happens in rate fetcher `start` method https://github.com/paraswap/paraswap-dex-lib/pull/370/files#diff-ba686099e8f3487fb4a1dd64ecb162d4036d10fce754697d05428bcdaae0b3c5R168.